setup.py

```diff
@@ -39,6 +39,7 @@
         "typer",
         "typing-extensions",
         "typing-inspect",
+        "wandb",
```

Comment:
there seems to be some panic, though not insurmountable around installing wandb. If I add it here, then they will just have it